README.md.template

```diff
@@ -351,7 +351,24 @@ $ solo node start
 
 # output is similar to example-1 
 ```
+## For Developer Working on Hedera Repo
 
+To set customized `settings.txt` file, edit the file
+`./resources/templates/settings.txt` before `node setup` command.
+
+To deploy node with local build hedera code, first run `./gradlew assemble` at the root directory of hedera repo, then
+  run the following command:
+```
+$ solo node setup --local-build-path node0=<path-to-hedera-repo>,node1=<path-to-hedera-repo>,node2=<path-to-hedera-repo>
+```
+To have nodes running with different versions of hedera application, please clone the repo in multiple directories,
+checkout to the respective version and build the code. Then run the above command
+and provide the path to the respective node.
+
+To deploy node with local build PTT code, run the following command:
+```
+$ solo node setup --local-build-path node0=<path-to-hedera-repo>,node1=<path-to-hedera-repo>,node2=<path-to-hedera-repo> --ptt-test-config PlatformTestingTool.jar,<path-to-test-json>
```

Comment:
As mentioned earlier, we need to offer a generalized flag for any other app and config. Something like below:
```
solo node setup --local-build-path ../hedera-services/platform-sdk/data --app PlatformTestingTool.jar --app-config path-to-test-json-1,path-to-test-json-2
```

Reply:
Updated the logic of parsing `--local-build-path` to support a default build path.
Also created separate flags for `app` and `app-config`